api/room/schema_query.py

```diff
@@ -264,3 +274,9 @@ def resolve_analytics_for_daily_room_events(self, info,
             )
 
         return all_days_events
+
+    @Auth.user_roles('Admin')
+    def resolve_notification_channels(self, info):
+        all_rooms = RoomModel.query.all()
```

Comment:
Good job @omobosteven. Though this query would give you all the room within the database, but I believe you'd only want the active rooms so I'd suggest you do this instead `all_rooms = RoomModel.query.filter_by(state="active")`. As it is you'll be getting also the archived rooms, with I don't believe would be getting any notifications.

Reply:
Thanks @C3real-kill3r for pointing that out.